package.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name": "frontend-boilerplate",
```

Comment:
@AniVerma17 Please change the name and description of the project.

Reply:
Done